middleware-catalog-module/catalog-model/src/main/java/co/tpg/catalog/model/Address.java

```diff
@@ -0,0 +1,67 @@
+package co.tpg.catalog.model;
+
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBAttribute;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBHashKey;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBTable;
+
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+/**
+ * Response class for Address entity
+ * 
+ * @author maxx
+ * @since 2019-10-11
+ */
+
+@Builder
+@Getter
+@Setter
+@ToString
+@DynamoDBTable(tableName = "Address")
+public class Address extends AbstractModel<String> {
```

Comment:
Add new lombok annotations in the top of the class: 
@NoArgsContructor
@AllArgsConstructor